server/lib/data/convert-users-json-to-csv.js

```diff
@@ -0,0 +1,22 @@
+const { parseAsync } = require('json2csv')
+
+function takeFirstPhoneNumber (item) {
```

Comment:
There shouldn't be a case when an ALB dataset has more than a single phone number as that is controlled via an upload through the web app. However, this makes sure only the first number is used and don't cause problems with the CSV dataset.